backend/src/main/java/com/fydp/backend/model/ChapterTextModel.java

```diff
@@ -7,6 +7,7 @@
 @Component
 public class ChapterTextModel {
     private Map<String, String> chpTxt;
+    private Map<String, Long> chpId;
```

Comment:
`chpIds`?

And maybe just change all the `chp`s to `chapter` for this file

AND rename this class to `ChapterTexts`? More obvious that it's for multiple chapters and removes the extra `Model` word

Reply:
Not sure if this class is needed at all. It is used in parseChapters and for some reason we are returning the extracted text (prior to summarization) back to frontend and I don't see why we are doing that. If the frontend doesnt need it we can just return a map of Chapter titles to summary ids.

Reply:
Oh right on, yea I think you should just clean this up so it doesn't return the raw text to the frontend then.

Reply:
done